### PR TITLE
WIP: cmd,core,eth,turbo: enable keeping multiple branches of block bodies

### PR DESCRIPTION
**Pull request by nanevardanyan**

### CODE REVIEW

File: core/rawdb/accessors_chain.go [hunk 1]
```diff
@@ -752,8 +751,15 @@ func MakeBodiesCanonical(tx kv.RwTx, from uint64, ctx context.Context, logPrefix
 		i := uint64(0)
 		if err := tx.ForAmount(kv.NonCanonicalTxs, hexutility.EncodeTs(bodyForStorage.BaseTxId+1), bodyForStorage.TxAmount-2, func(k, v []byte) error {
 			id := newBaseId + 1 + i
-			if err := tx.Put(kv.EthTx, hexutility.EncodeTs(id), v); err != nil {
-				return err
+			if transactionsV3 {
+				key := append(hexutility.EncodeTs(id), h.Bytes()...)
+				if err := tx.Put(kv.EthTxV2, key, v); err != nil {
```
Comment (AskAlexSharov) [inline core/rawdb/accessors_chain.go:756]: Let’s rename to V3 to align with ErigonV3

**Comment by AskAlexSharov** [comment]
See also TxLookup stage also - maybe it iterate over all of txs

File: core/rawdb/accessors_chain.go (hunk 1 above)
Comment (nanevardanyan) [reply core/rawdb/accessors_chain.go:756]: Yeah! Thanks, good point! will do.

**Comment by nanevardanyan** [comment]
Sure will take a look. So far I was looking for the places where the values get set in the table. Looks like there is only one place, and one in migration.

**Comment by AlexeyAkhunov** [comment]
I think this is a good start

**Comment by nanevardanyan** [comment]
Closing this one. Please refer to the new pr: https://github.com/ledgerwatch/erigon/pull/6754

**Pull request closed without merging**